client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -29,9 +30,16 @@ export default function Dashboard(): JSX.Element {
   }
 
   return (
-    <Grid container component="main" className={`${classes.root} ${classes.dashboard}`}>
+    <Box>
       <CssBaseline />
-      <NavBar loggedInUser={loggedInUser} />
-    </Grid>
+      <Box>
+        <NavBar loggedInUser={loggedInUser} />
+      </Box>
+      <Box className={classes.board}>
+        <KanbanProvider>
+          <Board />
+        </KanbanProvider>
+      </Box>
+    </Box>
```

Comment:
Re: the merge conflict here, I think you can just go ahead and overwrite what's in the main branch currently. I'll copy this into my local version and adapt my hover/dialog buttons to match this.

Reply:
I'll pull down the latest version. Probably should have done it before commiting.